lib/Cake/Cache/Cache.php

```diff
@@ -578,4 +578,41 @@ public static function remember($key, $callable, $config = 'default') {
 		return $results;
 	}
 
+/**
+ * Write data for key into a cache engine if it doesn't exist already.
+ *
+ * ### Usage:
+ *
+ * Writing to the active cache config:
+ *
+ * `Cache::add('cached_data', $data);`
+ *
+ * Writing to a specific cache config:
+ *
+ * `Cache::add('cached_data', $data, 'long_term');`
+ *
+ * @param string $key Identifier for the data
```

Comment:
Right, pretty much figured that by the time I submitted my comment xD I even capitalized "pecl" on the write() function for bonus points.
Thanks for the link. Should be fine now.